recipes/omniscidb-cuda/recipe/build.sh

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bash
+
+set -ex
+
+# Make sure -fPIC is not in CXXFLAGS (that some conda packages may
+# add), otherwise omniscidb server will crash when executing generated
+# machine code:
+export CXXFLAGS="`echo $CXXFLAGS | sed 's/-fPIC//'`"
+
+# Fixes https://github.com/Quansight/pearu-sandbox/issues/7
+#       https://github.com/omnisci/omniscidb/issues/374
+export CXXFLAGS="$CXXFLAGS -Dsecure_getenv=getenv"
+
+# Fixes `error: expected ')' before 'PRIxPTR'`
+export CXXFLAGS="$CXXFLAGS -D__STDC_FORMAT_MACROS"
+
+# resolves `fatal error: boost/regex.hpp: No such file or directory`
```

Comment:
Can you open an issue upstream about not using Boost_INCLUDE_DIRS in the nvcc command invocation?

Reply:
yes. But I suggest resolving this issue in feedstock as it make take time for upstream to fix it.

Reply:
Yes, that's fine working around this here, but please open an issue upstream.

Reply:
I have created a patch that resolves this issue as well as submitted the corresponding PR to upstream: https://github.com/omnisci/omniscidb-internal/pull/3984